azure-pipelines.yml

```diff
@@ -50,7 +50,7 @@ steps:
 
       git clone https://github.com/nanoframework/nanoFramework.Runtime.Native.git --depth 100 -b develop nanoFramework.Runtime.Native 
 
-      git clone https://github.com/nanoframework/nanoFramework.WebServer --depth 100 -b develop nanoFramework.WebServer 
+      git clone https://github.com/nanoframework/nanoFramework.WebServer --depth 105 -b develop nanoFramework.WebServer 
```

Comment:
build is failing because the last prior build was 101 commits ago - bumping this to see if it fixes the problem, then we can bump back down.